ciongke/Makefile

```diff
@@ -36,16 +39,18 @@ JENKINS_SECRET_FILE = ${HOME}/jenkins
 
 create-cluster:
 	gcloud -q container --project "$(PROJECT)" clusters create ciongke --zone "$(ZONE)" --machine-type "$(MACHINE_TYPE)" --scope "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.full_control","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management" --num-nodes "$(NUM_NODES)" --network "default" --enable-cloud-logging --enable-cloud-monitoring
-	kubectl create secret generic hmac-token --from-file=hmac=$(HOOK_SECRET_FILE)
-	kubectl create secret generic oauth-token --from-file=oauth=$(OAUTH_SECRET_FILE)
-	kubectl create secret generic jenkins-token --from-file=jenkins=$(JENKINS_SECRET_FILE)
-	make test-pr-image --no-print-directory
-	make hook-image --no-print-directory
-	make hook-deployment --no-print-directory
-	make hook-service --no-print-directory
+	@kubectl create secret generic hmac-token --from-file=hmac=$(HOOK_SECRET_FILE)
```

Comment:
does this automatically use the appropriate credentials from the container creation right before?